CONTRIBUTING.md

```diff
@@ -66,6 +66,29 @@ One of the best ways we can keep _fastlane_ an approachable, stable, and dependa
 
 Help us keep _fastlane_ open and inclusive. Please read and follow our [Code of Conduct][code of conduct].
 
+## Branding
+
+Although we don't have a formal style guide, we do have a few guidelines for how to refer to _fastlane_ and its tools:
```

Comment:
IMO this sentence is a little wordy and doesn't add much to mention lack of a style guide

```suggestion
We have a few guidelines for how to refer to _fastlane_ and its tools:
```